.github/workflows/system-tests.yml

```diff
@@ -96,80 +96,71 @@ jobs:
+        timeout-minutes: 5
         run: |
-          make
-        working-directory: neofs-s3-gw
+          make all
+        working-directory: neofs-node
 
       - name: Build neofs-s3-gw binaries
         run: |
-          make image
+          make all
         working-directory: neofs-s3-gw
 
-      - name: Add NeoFS S3-gw TAGs to s3-gw-test env config
-        run: |
-          sed -i -e 's/S3_GW_VERSION=_TAG_/S3_GW_VERSION=${{ env.CURRENT_TAG }}/' .github/testcases-env
-        working-directory: neofs-s3-gw
-
-      - name: Copy testcases-env file to .env for neofs-dev-env
-        run: |
-          cp .github/testcases-env ${GITHUB_WORKSPACE}/neofs-dev-env/.env
-        working-directory: neofs-s3-gw
-
-################################################################
+      - name: Download latest stable neofs-rest-gw
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-rest-gw'
+          version: 'tags/v0.7.2'
+          file: 'neofs-rest-gw-linux-amd64'
+          target: 'neofs-testcases/neofs-rest-gw'
 
-      - name: Prepare hosts
-        timeout-minutes: 5
-        run: |
-          make get
-          sudo ./bin/update_hosts.sh
-          sudo chmod a+w vendor/hosts
-        working-directory: neofs-dev-env
+      - name: Download latest stable neo-go
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neo-go'
+          version: 'tags/v0.104.0'
+          file: 'neo-go-linux-amd64'
+          target: 'neofs-testcases/neo-go'
 
       - name: Prepare venv
+        id: prepare_venv
         timeout-minutes: 30
         run: |
-          make venv.local-pytest
+          make venv.pytest
+          echo "$(pwd)" >> $GITHUB_PATH
         working-directory: neofs-testcases
 
-      - name: Prepare Dev-Env to run tests
-        id: prepare_test_env
+      - name: Copy built binaries
         timeout-minutes: 30
         run: |
-          make prepare-test-env
-          echo "$(pwd)/vendor" >> $GITHUB_PATH
-        working-directory: neofs-dev-env
+          cp bin/* ${GITHUB_WORKSPACE}/neofs-testcases/
+        working-directory: neofs-node
 
-      - name: Copy neofs-s3-gw binaries to vendor directory
+      - name: Copy neofs-s3-gw binaries to testcases directory
         run: |
-          cp bin/* ${GITHUB_WORKSPACE}/neofs-dev-env/vendor/
+          cp bin/* ${GITHUB_WORKSPACE}/neofs-testcases/
         working-directory: neofs-s3-gw
 
-################################################################
+      - name: Chmod binaries
+        run: |
+          sudo chmod a+x neofs-cli
+          sudo chmod a+x neofs-adm
+          sudo chmod a+x neofs-ir
+          sudo chmod a+x neofs-lens
+          sudo chmod a+x neofs-node
+          sudo chmod a+x neofs-rest-gw
+          sudo chmod a+x neo-go
+          sudo chmod a+x neofs-s3-authmate
+          sudo chmod a+x neofs-s3-gw
+        working-directory: neofs-testcases
+
```

Comment:
Maybe, it makes sense to `chmod` binaries before copying? I mean afters rows 137, 142 make `chmod -r a+x ./bin`
and then copy updated binaries?
I don't insist, just an another option

Reply:
The problem is that binaries come from different places, so it is better to `chmod` them in one place, just to be sure nothing is left.